spec/support/authentication_spec_helper.rb

```diff
@@ -14,7 +14,7 @@ def agent_auth_hash_from_inclusion_connect(agent, inclusion_connect_token_id)
       origin: "inclusion_connect",
       signature: agent.sign_with(timestamp),
       created_at: timestamp,
-      inclusion_connect_token_id:
+      "inclusion_connect_token_id" => inclusion_connect_token_id
```

Comment:
La session dans les tests ne se comporte pas comme en prod.
En prod les symbol [sont convertis](https://stackoverflow.com/questions/23530055/ruby-on-rails-sneakily-changing-nested-hash-keys-from-symbols-to-strings) par le fait que session est un objet ActionDispatch::Request::Session
Dans les tests la session est un hash tout simple et donc on a besoin de convertir la clé ici car on fait un dig dans le contrôleur : `session.dig("agent_auth", "inclusion_connect_token_id")`
J'ai préféré résoudre ce problème de cette facon en attendant de faire en sorte que les sessions dans les tests reflètent la prod (en convertissant toutes les clés en string ? en initialisant correctement les objets session ?)

Reply:
Du coup ma remarque précédente devrait aussi régler ce problème il me semble

Reply:
c'est le cas, parfait :)